src/Time/TimeStamp.hs

```diff
@@ -10,19 +11,38 @@ module Time.TimeStamp
        , timeDiff
        , timeAdd
        , timeMul
+       , (*:*)
        , timeDiv
+       , (/:/)
+
+         -- * Other operators
+       , (+:+)
+       , (-:-)
+       , (-%-)
+
        ) where
 
-import Time.Rational (KnownRat, Rat, RatioNat)
-import Time.Units (Time (..))
+import Time.Rational (KnownDivRat, KnownRat, Rat, RatioNat)
+import Time.Units (Time (..), toUnit)
+
+-- $setup
+-- >>> import Time.Units (Second, minute, sec)
 
 -- | Similar to 'Time' but has no units and can be negative.
 newtype TimeStamp = TimeStamp Rational
     deriving (Show, Read, Num, Eq, Ord, Enum, Fractional, Real, RealFrac)
 
 
--- | Returns the result of comparison of two 'Timestamp's and
--- the 'Time' of that difference of given time unit.
+{- | Returns the result of comparison of two 'Timestamp's and
+the 'Time' of that difference of given time unit.
+
+>>> timeDiff @Second (TimeStamp 4) (TimeStamp 2)
+(GT,2s)
+
+>>> timeDiff @Second (TimeStamp 2) (TimeStamp 4)
```

Comment:
Would be also good to add example with different time units, like, `Minute` or `Hour`